shell/platform/windows/flutter_windows_engine.h

```diff
@@ -170,6 +170,9 @@ class FlutterWindowsEngine {
   // given |texture_id|.
   bool MarkExternalTextureFrameAvailable(int64_t texture_id);
 
+  // Posts the given callback onto the raster thread.
+  bool PostRasterThreadTask(std::function<void()> callback);
```

Comment:
This should be named `PostRenderThreadTask` for consistency with the embedder API.